src/routes/home/Home.js

```diff
@@ -431,6 +458,34 @@ class Home extends React.Component {
       formatted_address: 'Finding Address...',
       addressProvenance,
     });
+
+    const {
+      state: { boroughBoundariesFeatureCollection },
+    } = this;
+    // TODO the above/below blocks were copied/adapted from ElectriCitibikes.js, consider unifying them
+    if (boroughBoundariesFeatureCollection) {
+      console.time('new PolygonLookup'); // eslint-disable-line no-console
+      const lookup = new PolygonLookup(boroughBoundariesFeatureCollection);
+      console.timeEnd('new PolygonLookup'); // eslint-disable-line no-console
+
+      const end = { latitude, longitude };
+      const BoroName = getBoroNameMemoized({ lookup, end });
```

Comment:
this was copied from ElectriCitibikes.js, consider unifying them

Reply:
no longer needed, as of https://github.com/josephfrazier/reported-web/pull/410/commits/d8938cfb9f9aa2c1962022b38c4575ce6eb104cb